EmailNotifications/Email.cs

```diff
@@ -0,0 +1,118 @@
+﻿using System;
+using System.ComponentModel;
+using System.Net.Mail;
+using System.Web;
+using System.Net;
+using Microsoft.Extensions.Configuration;
+using System.Threading.Tasks;
+using System.IO;
+
+namespace EmailNotifications
+{
+    public class Email
+    {
+        static bool mailSent = false;
+
+        string[] values;
+        public Email(string[] emailValue)
+        {
+            values = emailValue;
+
+        }
+        private static void SendCompletedCallback(object sender, AsyncCompletedEventArgs e)
+        {
+            // Get the unique identifier for this asynchronous operation.
+            String token = (string)e.UserState;
+
+            if (e.Cancelled)
+            {
+                Console.WriteLine("[{0}] Send canceled.", token);
+            }
+            if (e.Error != null)
+            {
+                Console.WriteLine("[{0}] {1}", token, e.Error.ToString());
+            }
+            else
+            {
+                Console.WriteLine("Message sent.");
+            }
+            mailSent = true;
+        }
+
+  
+        public void sendEmail()
+        {
+            string[] styleProperties = { "color:red;padding-top: 40px;font-weight: bold", "color:red;font-weight:bold", "color:black;padding-top: 11px",
```

Comment:
2 part comment: 
1-  why are you declaring CSS in here. 
2- These need to be structured better, I have dm'd you a possible solution.

Reply:
done